numpy/core/src/umath/ufunc_object.c

```diff
@@ -319,6 +319,23 @@ _find_array_prepare(ufunc_full_args args,
     NPY_ITER_NO_BROADCAST | \
     NPY_ITER_NO_SUBTYPE | \
     NPY_ITER_OVERLAP_ASSUME_ELEMENTWISE
+
+/* Called at module initialization to set the matmul ufunc output flags */
```

Comment:
It would be worth expanding this comment to explain:
* what about the defaults is insufficient
* that in general, the default output flags are not suitable for most gufuncs

Reply:
adding